kvirt/k3s/__init__.py

```diff
@@ -72,11 +73,21 @@ def create(config, plandir, cluster, overrides):
             else:
                 error("You need to define api_ip in your parameters file")
                 os._exit(1)
-    install_k3s_channel = data.get('install_k3s_channel', 'stable')
-    if install_k3s_channel not in ['stable', 'latest', 'testing']:
-        error("Invalid K3S install channel %s" % install_k3s_channel)
-        os._exit(1)
     data['basedir'] = '/workdir' if os.path.exists("/i_am_a_container") else '.'
+    install_k3s_args = []
+    for arg in data:
+        if arg.startswith('install_k3s'):
```

Comment:
And all the K3s install args do not need to be defined in the `kcli_default.yml` file?

Reply:
nope, it s up to the user to specify what they want, and the benefit is that, assuming that this nomenclature keeps beeing around, newer parameters will be available without changes needed

Reply:
What's not to like!